app/views/proposal/_encouragement_text.html.haml

```diff
@@ -0,0 +1,23 @@
+%p.lead
+  - if @conference.event_types.any?
+    You can submit proposals for
+    = "#{event_types(@conference)}."
+  - if @conference.tracks.any?
+    Proposals should fit in one of the
+    = "#{pluralize(@conference.tracks.count, 'track')}:"
+    = "#{tracks(@conference)}."
+  - if @conference.call_for_paper
```

Comment:
I think the ability should only refer to :new, :create actions. 
Why not access your proposals index page whenever? Before cfp, after cfp... 

We should also make the 'New Proposal' button only appear accordingly.
This view could still stay as it is I guess.